R/tags.R

```diff
@@ -383,6 +474,9 @@ throw_if_tag_function <- function(tag) {
 #'   normally be written around this tag. Valid options include `before`,
 #'   `after`, `outside`, `after-begin`, and `before-end`.
 #'   Any number of these options can be specified.
+#' @param .render A function that is called during render time. This function
```

Comment:
Part of me wanted this to be `.onRender` because of the precedence in `htmlwidgets::createWidget()`, but I don't feel super strongly about it